src/Components/Header.js

```diff
@@ -89,6 +94,11 @@ class Header extends Component {
                     SETTINGS
                   </a>
                 </div>
+                <div className="sideNavLink">
+                  <a href='/instructions'>
```

Comment:
i know using these `href` links "legit" html links has been problematic working with React Router for some teams.  The maintainers of React Router recommend this package to handle people clicking on links:

https://github.com/react-bootstrap/react-router-bootstrap

Give it a shot :)